React/src/components/FormControl/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useForm } from 'react-hook-form';
+
+import { RegisterUser } from 'interfaces/RegisterUser';
+
+import styles from './styles.module.scss';
+
+export function FormControl() {
+  const { register, handleSubmit } = useForm<RegisterUser>();
+
+  return (
+    <div className={styles.pageContainer}>
+      <div className={styles.loginContainer}>
```

Comment:
This divs are the same that the ones on the AuthComponent, review if are neccessary.